{"name": "Проверка перехода в профиль по клику на \"Личный кабинет\" в хэдере", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16", "trace": "self = <test_account_page.TestAccountPage object at 0x107217230>\ndriver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"6dace3ce-91be-4b9c-a486-ee3a6edf6f00\")>, set_user_tokens = None\n\n    @allure.title('Проверка перехода в профиль по клику на \"Личный кабинет\" в хэдере')\n    def test_navigate_to_account_page_success(self, driver, set_user_tokens):\n        main_page = MainPage(driver)\n        account_page = AccountPage(driver)\n        main_page.click_on_personal_account_in_header()\n>       account_page.wait_visibility_of_description()\n\ntests/test_account_page.py:14: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npage_objects/account_page.py:17: in wait_visibility_of_description\n    self.wait_visibility_of_element(AccountPageLocators.description_of_section)\npage_objects/base_page.py:13: in wait_visibility_of_element\n    WebDriverWait(self.driver, 10).until(expected_conditions.visibility_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"6dace3ce-91be-4b9c-a486-ee3a6edf6f00\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x107230a40>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\nE       dom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException"}, "steps": [{"name": "Кликнуть по кнопке перехода в личный кабинет в хэдере", "status": "passed", "steps": [{"name": "Подождать прогрузки элемента", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', '//p[text()=\"Личный Кабинет\"]/parent::a')"}], "start": 1726427086444, "stop": 1726427086455}, {"name": "Кликнуть на элемент", "status": "passed", "steps": [{"name": "Проверить кликабельность элемента", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', '//p[text()=\"Личный Кабинет\"]/parent::a')"}], "start": 1726427086455, "stop": 1726427086465}], "parameters": [{"name": "locator", "value": "('xpath', '//p[text()=\"Личный Кабинет\"]/parent::a')"}], "start": 1726427086455, "stop": 1726427086722}], "start": 1726427086443, "stop": 1726427086722}, {"name": "Подождать прогрузки текста описания раздела", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/kosova.ea/Diplom/Diplom_3/page_objects/account_page.py\", line 17, in wait_visibility_of_description\n    self.wait_visibility_of_element(AccountPageLocators.description_of_section)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/kosova.ea/Diplom/Diplom_3/page_objects/base_page.py\", line 13, in wait_visibility_of_element\n    WebDriverWait(self.driver, 10).until(expected_conditions.visibility_of_element_located(locator))\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Подождать прогрузки элемента", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/kosova.ea/Diplom/Diplom_3/page_objects/base_page.py\", line 13, in wait_visibility_of_element\n    WebDriverWait(self.driver, 10).until(expected_conditions.visibility_of_element_located(locator))\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "parameters": [{"name": "locator", "value": "('xpath', '//p[contains(@class, \"Account_text\")]')"}], "start": 1726427086722, "stop": 1726427096860}], "start": 1726427086722, "stop": 1726427096866}], "parameters": [{"name": "driver", "value": "<class 'selenium.webdriver.firefox.webdriver.WebDriver'>"}], "start": 1726427086443, "stop": 1726427096868, "uuid": "c67c063c-1b0b-4f65-aef6-e708db9ea275", "historyId": "69a329a001ffff117081d1b6f47b5770", "testCaseId": "fa73ae3771d826c24d9608176601ee82", "fullName": "tests.test_account_page.TestAccountPage#test_navigate_to_account_page_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_account_page"}, {"name": "subSuite", "value": "TestAccountPage"}, {"name": "host", "value": "NB-ISC-69172.local"}, {"name": "thread", "value": "15735-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_account_page"}]}